src/core/updateWebpackConfig.js

```diff
@@ -13,31 +13,25 @@ const updateWebpackConfig = (webpackConfig, env, serverInfo) => {
     {
       test: /\.jsx?$/,
       include: srcFolder,
-      loader: 'babel',
+      loader: 'babel-loader',
       query: {
+        babelrc: false,
         presets: [
           'es2015',
           'react',
         ],
```

Comment:
I think it’s mostly explained here: https://react-styleguidist.js.org/docs/development.html

Eventually you’ll have a conflict with user’s loaders: you either break their code or they break yours.